py/selenium/webdriver/common/service.py

```diff
@@ -160,14 +160,9 @@ def stop(self):
                     except AttributeError:
                         pass
                 self.process.terminate()
-                self.process.kill()
                 self.process.wait()
+                self.process.kill()
                 self.process = None
         except OSError:
             # kill may not be available under windows environment
             pass
-
-    def __del__(self):
-        # subprocess.Popen doesn't send signal on __del__;
-        # we have to try to stop the launched process.
-        self.stop()
```

Comment:
yea, i think that just swallowing exceptions from a try/except is ok here.  

I've updated my branch and re-added the __del__  

p.s. I'll get that dastardly `__del__` out of there eventually.... fear not! :)

p.p.s. sorry I also piggybacked the tiny style change to the connection test in `send_remote_shutdown_command`...  since it was so egresiusly un-pythonic. :)